vinyldns/vinyldns
# remove empty record data fields from json

![Screen Shot 2019-10-18 at 1 48 52 PM](https://user-images.githubusercontent.com/4439228/67116305-2b071a00-f1ae-11e9-8f56-0232003e5523.png)

Changes in this pull request:
- In the DNS Change form in the portal if there's an entry with a changeType of "DeleteRecordSet" and the record data value is undefined or an empty string we will delete the record data attributes before sending the JSON to the API.

To test this:
- Choose "DeleteRecordSet" change type, enter data in the Input Name and Record Data fields.
- Delete the data in the Record Data field
- Submit the DNS Change

The DNS Change should not return an error related invalid record data
